.php_cs.dist

```diff
@@ -111,7 +111,7 @@ return Config::create()
         'phpdoc_align' => false,
         'phpdoc_summary' => false,
         'php_unit_set_up_tear_down_visibility' => true,
-        'php_unit_strict' => true,
+        'php_unit_strict' => false,
```

Comment:
I would avoid it. There is always a possibility to rewrite tests to use `assertSame` instead of `assertEquals`. Do we need to decrease the strict level just because of 1 assertion from thousand of tests?

Reply:
>There is always a possibility to rewrite tests to use assertSame instead of assertEquals

How do you want to compare two objects by their values then? (in a sane way)

Reply:
[Here](https://github.com/infection/infection/pull/752/files#diff-949f564bd26d117d5334b0c8ff652cccR79)?

There are many ways to do it (including converting objects to arrays, checking objects by json schema, checking only some of the fields, etc. etc. etc.). 

For the simplicity for this particular example, I would say doing this will be enough:

```php
$this->assertTrue($container['regular service']->regular, 'Error message...');
```